lean4-infoview/src/infoview/messages.tsx

```diff
@@ -99,7 +99,12 @@ export function AllMessages({uri: uri0}: { uri: DocumentUri }) {
             try {
                 return await getInteractiveDiagnostics(rs, { uri: uri0, line: 0, character: 0 }) || [];
             } catch (err: any) {
-                console.log('getInteractiveDiagnostics error ', err)
+                if (err?.code === -32801) {
+                    // Document has been changed since we made the request.
+                    // This can happen while typing quickly, so server will catch up on next edit.
```

Comment:
I would also explain the infoview codepath:
```suggestion
                    // Document has been changed since we made the request. This can happen
                    // while typing quickly. When the server catches up on next edit, it will
                    // send new diagnostics to which the infoview responds by calling
                    // `getInteractiveDiagnostics` again.
```